Add INT-P4 SW component

rafagarciaUMA opened the pull request:
Add first complete version INT-P4 SW component from 'int_p4_sw' branch to 'develop' branch.

jmaciasportela commented on changelog.md:
```diff
@@ -5,6 +5,7 @@
 - New component `upf_p4_sw`
```
You need to put a version number that will be tracked by the global version

rafagarciaUMA commented:
Hi Jesús,
I added a new commit and I added version 1.0.0 to the changelog. With it, would it be? Or does something else need to be modified?
Thanks for the review
